ci/cron/src/Main.hs

```diff
@@ -183,12 +184,20 @@ fetch_gh_paginated url = do
                 (_, _, _, [url, rel]) -> (rel, url)
                 _ -> fail $ "Assumption violated: link header entry did not match regex.\nEntry: " <> l
 
-data PreVersion = PreVersion { prerelease :: Bool, tag :: Version }
-instance JSON.FromJSON PreVersion where
-    parseJSON = JSON.withObject "PreVersion" $ \v -> PreVersion
-        <$> v JSON..: "prerelease"
-        <*> let json_text = v JSON..: "tag_name"
-            in (version . Text.tail <$> json_text)
+data GitHubRelease = GitHubRelease { prerelease :: Bool, tag :: Version, assets :: [String] }
```

Comment:
I would recommend to go directly for a `URI` here instead of a `String` to make it clear what you are storing.
EDIT: Or the `Asset` type that I suggested below.

Reply:
I assume you mean [this one](https://hackage.haskell.org/package/network-uri-2.6.3.0/docs/Network-URI.html)?

Reply:
Yep, sorry should have qualified that.